### ISSUE TITLE
Question about reproducing progressive prompt T5 experiments

### ISSUE THREAD

**Issue by dahunshin** [arazd/ProgressivePrompts#5]
Dear author,

I recently read your paper titled “Progressive Prompts: Continual Learning for Language Models” with great interest. I was particularly impressed by the excellent results of the T5-large experiments, which motivated me to attempt reproducing them.
Following the instructions provided in your public Git repository, I set up a virtual environment and conducted the experiments using the command below, based on the experimental details in Appendix A.4 of your paper. However, I was unable to achieve the accuracy reported in the paper. My results yielded an average accuracy of around 63%, whereas the paper reports an accuracy of 75.2%. I suspect there might be an error in the command. Could you please help me identify it?
"""
python train_t5_cl.py --task_list dbpedia_14 amazon yahoo_answers_topics ag_news
 --select_k_per_class 64 --lr 0.3 --batch_size 8 --num_epochs 10 --freeze_weights 1 --prefix_len 50 --model_name t5-large --early_stopping 1 --data_replay_freq 10 --save_name T5_experiment --save_dir my_path_to_save_directory
"""
Thank you for your time and assistance.

**Comment by chen-fei-fan** [comment]
> Dear author,
> 
> I recently read your paper titled “Progressive Prompts: Continual Learning for Language Models” with great interest. I was particularly impressed by the excellent results of the T5-large experiments, which motivated me to attempt reproducing them. Following the instructions provided in your public Git repository, I set up a virtual environment and conducted the experiments using the command below, based on the experimental details in Appendix A.4 of your paper. However, I was unable to achieve the accuracy reported in the paper. My results yielded an average accuracy of around 63%, whereas the paper reports an accuracy of 75.2%. I suspect there might be an error in the command. Could you please help me identify it? """ python train_t5_cl.py --task_list dbpedia_14 amazon yahoo_answers_topics ag_news --select_k_per_class 64 --lr 0.3 --batch_size 8 --num_epochs 10 --freeze_weights 1 --prefix_len 50 --model_name t5-large --early_stopping 1 --data_replay_freq 10 --save_name T5_experiment --save_dir my_path_to_save_directory """ Thank you for your time and assistance.
对啊 我也是 在order1上运行 四个任务平均下来也就69.0

**Comment by cff41** [comment]
> > 尊敬的作者：
> > 我最近饶有兴趣地阅读了您题为“渐进式提示：语言模型的持续学习”的论文。其中，T5-large 实验的出色结果给我留下了深刻的印象，这促使我尝试复现这些结果。我按照您公开的 Git 仓库中的说明，搭建了一个虚拟环境，并根据您论文附录 A.4 中的实验细节，使用以下命令进行了实验。然而，我未能达到论文中报告的准确率。我的结果平均准确率约为 63%，而论文中报告的准确率为 75.2%。我怀疑命令中可能存在错误。您能否帮我找出问题所在？ """ python train_t5_cl.py --task_list dbpedia_14 amazon yahoo_answers_topics ag_news --select_k_per_class 64 --lr 0.3 --batch_size 8 --num_epochs 10 --freeze_weights 1 --prefix_len 50 --model_name t5-large --early_stopping 1 --data_replay_freq 10 --save_name T5_experiment --save_dir my_path_to_save_directory """ 谢谢你的时间和帮助。
> > 对啊 我也在order1上运行 四个任务平均下来大约69.0

而且你的每个类别选的训练数据也很多 你还加入了回放机制
